From: Noah Meyerhans <frodo@morgul.net>
Date: Fri, 24 Jan 2025 18:12:26 -0500
Subject: don't build setup or extension components

---
 Cargo.toml | 2 --
 1 file changed, 2 deletions(-)

Index: GuestProxyAgent/Cargo.toml
===================================================================
--- GuestProxyAgent.orig/Cargo.toml
+++ GuestProxyAgent/Cargo.toml
@@ -5,8 +5,6 @@ resolver = "2"
 members = [
     "proxy_agent_shared",
     "proxy_agent",
-    "proxy_agent_extension",
-    "proxy_agent_setup",
 ]
 
 exclude = [
