From: Noah Meyerhans <noahm@debian.org>
Date: Wed, 15 Jan 2025 20:23:12 +0000
Subject: [PATCH 5/5] loosen tokio-util constraint

---
 proxy_agent/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/proxy_agent/Cargo.toml b/proxy_agent/Cargo.toml
index 1315743..a3461a0 100644
--- a/proxy_agent/Cargo.toml
+++ b/proxy_agent/Cargo.toml
@@ -18,7 +18,7 @@ hmac-sha256 = "1.1.6"         # use HMAC using the SHA-256 hash function
 hex = "0.4.3"                 # hex encode 
 regex = "1.9.5"               # match process name in cmdline
 tokio = { version = "1", features = ["rt", "rt-multi-thread", "time",  "net", "macros", "sync"] }
-tokio-util = "0.7.11"
+tokio-util = "0.7"
 http = "1.1.0"
 http-body-util = "0.1"
 hyper = { version = "1", features = ["server", "http1", "client"] }
