From: Noah Meyerhans <noahm@debian.org>
Date: Wed, 15 Jan 2025 17:27:44 +0000
Subject: [PATCH 2/5] Relax serde-xml-rs version requirement

---
 proxy_agent/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/proxy_agent/Cargo.toml b/proxy_agent/Cargo.toml
index b38bc4a..3145a18 100644
--- a/proxy_agent/Cargo.toml
+++ b/proxy_agent/Cargo.toml
@@ -12,7 +12,7 @@ once_cell = "1.17.0"          # use Lazy
 serde = "1.0.152"
 serde_derive = "1.0.152"
 serde_json = "1.0.91"         # json Deserializer
-serde-xml-rs = "0.6.0"        # xml Deserializer
+serde-xml-rs = "0.5.0"        # xml Deserializer
 bitflags = "2.6.0"            # support bitflag enum
 hmac-sha256 = "1.1.6"         # use HMAC using the SHA-256 hash function
 hex = "0.4.3"                 # hex encode 
