From e31e66f312df8de7719e84774adff471a1442d32 Mon Sep 17 00:00:00 2001
From: Noah Meyerhans <noahm@debian.org>
Date: Wed, 15 Jan 2025 20:21:33 +0000
Subject: [PATCH 4/5] use newer itertools

---
 proxy_agent/Cargo.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: GuestProxyAgent/proxy_agent/Cargo.toml
===================================================================
--- GuestProxyAgent.orig/proxy_agent/Cargo.toml
+++ GuestProxyAgent/proxy_agent/Cargo.toml
@@ -7,7 +7,7 @@ build = "build.rs"
 
 [dependencies]
 proxy_agent_shared = { path ="../proxy_agent_shared"}
-itertools = "0.10.5"          # use to sort iterator elements into a new iterator in ascending order
+itertools = "0.13"          # use to sort iterator elements into a new iterator in ascending order
 once_cell = "1.17.0"          # use Lazy
 serde = "1.0.152"
 serde_derive = "1.0.152"
@@ -69,4 +69,4 @@ assets = [
     ["azure-proxy-agent", "usr/sbin/azure-proxy-agent", "755"],  # Binary
     ["proxy-agent.json", "etc/azure/proxy-agent.json", "644"],
     ["ebpf_cgroup.o", "usr/lib/azure-proxy-agent/ebpf_cgroup.o", "644"],
-]
\ No newline at end of file
+]
